src/ScaleformMCM.cpp

```diff
@@ -195,9 +205,10 @@ namespace ScaleformMCM {
 		virtual void Invoke(Args* args) {
 			args->result->SetBool(false);
 
-			if (args->numArgs < 2) return;
+			if (args->numArgs < 3) return;
 			if (args->args[0].GetType() != GFxValue::kType_String) return;
 			if (args->args[1].GetType() != GFxValue::kType_String) return;
+			if (args->args[2].GetType() != GFxValue::kType_String) return;
```

Comment:
Please update the function signature in the comments if you've changed the order of the arguments.